frog/imports/ui/TeacherView/ButtonList.jsx

```diff
@@ -31,7 +47,11 @@ const ButtonList = ({
     {
       states: ['STARTED'],
       type: 'primary',
-      onClick: () => nextActivity(session._id),
+      onClick: () => {
+        updateSessionCountdownStartTime(session._id, -1);
+        updateSessionCountdownTimeLeft(session._id, DEFAULT_TIME_LEFT);
```

Comment:
Another thought, remember that a variable in Sessions collection can be an object, so you could have

type definition: 
countdown: {state: 'none'} | {state: 'paused', endTime: number}, {state: 'running', endTime: number} 
or something similar. The idea is to "outlaw illegal states"... Having to keep track of two different variables can easily bring bugs.